contrib/snap/snapcraft.yaml

```diff
@@ -0,0 +1,66 @@
+name: dogecoin-core # you probably want to 'snapcraft register <name>'
+version: '1.14.3'
```

Comment:
If we're merging this into 1.14.4-dev, should we name this differently, eg, `1.14.4-pre` or something like that?

Reply:
Ah, I don't think I was clear enough in my explanation. This only works for actual releases because it grabs the releases from the [releases page](https://github.com/dogecoin/dogecoin/releases), where there are files named `dogecoin-1.14.3-...`. If we intended to release this in 1.14.4, it would need to be versioned 1.14.4 and would only start working when the 1.14.4 release is out.